cache/src/test/java/io/envoyproxy/controlplane/cache/v2/SnapshotTest.java

```diff
@@ -127,21 +127,22 @@ public void resourcesReturnsExpectedResources() {
     // We have to do some lame casting to appease java's compiler, otherwise it fails to compile due to limitations with
     // generic type constraints.
 
-    assertThat((Map<String, Message>) snapshot.resources(CLUSTER_TYPE_URL))
-        .containsEntry(CLUSTER_NAME, CLUSTER)
-        .hasSize(1);
-
-    assertThat((Map<String, Message>) snapshot.resources(ENDPOINT_TYPE_URL))
-        .containsEntry(CLUSTER_NAME, ENDPOINT)
-        .hasSize(1);
-
-    assertThat((Map<String, Message>) snapshot.resources(LISTENER_TYPE_URL))
-        .containsEntry(LISTENER_NAME, LISTENER)
-        .hasSize(1);
-
-    assertThat((Map<String, Message>) snapshot.resources(ROUTE_TYPE_URL))
-        .containsEntry(ROUTE_NAME, ROUTE)
-        .hasSize(1);
+    // todo: come back here
```

Comment:
Should be fixex before merge

Reply:
Sorry, this review slipped through my email. This should be an easy fix

Reply:
Fix is easy, but looks like I don't have commit privileges to this repo, and my fork of it is based on https://github.com/sschepens/java-control-plane/tree/delta-xds and hence very out of date. @slonka is it possible to get me privileges on this repo? Or this is a very small fix, should just be a matter of  substituting this for the commented out code:
```
     assertThat(snapshot.resources(CLUSTER_TYPE_URL))
        .containsEntry(CLUSTER_NAME, VersionedResource.create(CLUSTER))
        .hasSize(1);

    assertThat(snapshot.resources(ENDPOINT_TYPE_URL))
        .containsEntry(CLUSTER_NAME, VersionedResource.create(ENDPOINT))
        .hasSize(1);

    assertThat(snapshot.resources(LISTENER_TYPE_URL))
        .containsEntry(LISTENER_NAME, VersionedResource.create(LISTENER))
        .hasSize(1);

    assertThat(snapshot.resources(ROUTE_TYPE_URL))
        .containsEntry(ROUTE_NAME, VersionedResource.create(ROUTE))
        .hasSize(1);
```